triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -1606,13 +1606,13 @@ object Runner {
       case Trigger.Level.High =>
         val result = for {
           inFlightCommands <- getInFlightCommands(svalue)
-          commands <- mapLookup(SText(commandId), inFlightCommands)
+          commands <- mapLookup(commandId, inFlightCommands)
```

Comment:
Bug fix: keys are records with unpacks - not plain Text command IDs

Reply:
At the Scala level, we're only working with SValue's, so the call type checks fine.

As the level of Speedy, we're never hitting the engine here, so Speedy never has a chance to perform any type checking - all manipulation here is done with raw SValue's.